### DISCUSSION TITLE
Server "crashed" / stays at 1-3% CPU load (~5-10% normal)

### DISCUSSION THREAD

**Discussion by VooDooCyberCookie** [lloesche/valheim-server-docker#49] [Q&A] [answered]
Hey folks,

I just experience a strange thing.

I was logged into the game on my own server.
All of a sudden I got disconnected. Trying to reconnect ended up with "Inkompatible Version".
I checked my monitory and noticed the container CPU load droped to almost 0% (pending at 1-3% while its usually at around 10%).
Further I am not able to get the server running again. Restoring backups or even removing the world and get a fresh one does not work. The container start up looks fine. But after the CPU spike for start-up it always drops to 1-3% CPU load (I guess at this state the game is not actually running - leading to "Inkompatible Version".

I've also noticed stange backup size's right before the crash:
```
4.8M Feb 16 22:50 worlds-20210216-215034.zip
4.8M Feb 16 23:50 worlds-20210216-225034.zip
5.0M Feb 17 00:50 worlds-20210216-235035.zip
5.2M Feb 17 01:50 worlds-20210217-005035.zip
5.4M Feb 17 02:50 worlds-20210217-015036.zip
5.4M Feb 17 03:50 worlds-20210217-025037.zip
5.5M Feb 17 04:50 worlds-20210217-035037.zip
5.5M Feb 17 05:50 worlds-20210217-045038.zip
5.5M Feb 17 06:50 worlds-20210217-055039.zip
5.5M Feb 17 07:50 worlds-20210217-065040.zip
5.5M Feb 17 08:50 worlds-20210217-075040.zip
5.5M Feb 17 09:50 worlds-20210217-085041.zip
5.5M Feb 17 10:50 worlds-20210217-095042.zip
5.5M Feb 17 11:50 worlds-20210217-105043.zip
 11M Feb 17 11:59 worlds-20210217-105901.zip       <===
 164 Feb 17 12:08 worlds-20210217-110804.zip        <===
 164 Feb 17 12:12 worlds-20210217-111214.zip        <===
 11M Feb 17 12:20 worlds-20210217-112000.zip       <===
4.9M Feb 17 12:36 worlds-20210217-113648.zip
```
Doesn't matter which backup I choose, I end up as described above.

**Comment by lloesche** [comment, marked as answer]
There was a Valheim update today. Have you updated your local version in Steam? The server will always run the latest version. "Incompatible Version" just means that the version of the server is newer than the version on your PC.

**Comment by VooDooCyberCookie** [comment]
Oh my god... this is embarrassing :D ..I even checked that but it seems like I was to fast (client update schedule took maybe few minutes to check for the update...)

Thank you!

**Comment by lloesche** [comment]
🙂 no worries. This might be helpful for others who run into the same issue. I'll convert it to a discussion.

**Comment by LHolst** [comment]
in my case the `updater` process did shutdown the `valheim-server`-process gracefully. restartet new version successfully 

i noticed that the `world-xxx.zip` file now contains the `.old` files in addition to the `.fw`/`.db` files

```
Archive: config/backups/worlds-XXXXX.zip
  Length      Date    Time    Name
---------  ---------- -----   ----
        0  2021-02-17 13:39   worlds/
       47  2021-02-17 13:19   worlds/Valheimworld.fwl.old
       47  2021-02-17 13:39   worlds/Valheimworld.fwl
 22147978  2021-02-17 13:39   worlds/Valheimworld.db
 22139506  2021-02-17 13:19   worlds/Valheimworld.db.old
---------                     -------
 44287578                     5 files
```
